### ISSUE TITLE
Indent Guides conditions + rules

### ISSUE THREAD

**Issue by nandsoar** [microsoft/vscode#183735]
[Feature] We can specify where indentation lines are rendered (e.g. only show indentation lines for function scopes, and omit them everywhere else). In this example it would be useful for a better view of all the functions at a glance with minimal distraction, but we can customize it or just select a different preset rule for how the indents are rendered.

Mainly I wanted it to feel more like how Visual Studio does indent guides.

**Comment by IllusionMH** [comment]
Have you checked bracket guides settings? Examples are here

https://code.visualstudio.com/updates/v1_67#_bracket-guide-improvements

**Comment by eskevv** [comment]
> Have you checked bracket guides settings? Examples are here
> 
> https://code.visualstudio.com/updates/v1_67#_bracket-guide-improvements 

I don't think that solves what I'm trying to request here. Those seem to be just horizontal guides for all indentation level.

**Comment by IllusionMH** [comment]
Horizontal part is just an example I was able to quickly find from phone, you should be able to show all vertical (only) brackets guides.
